variables.tf

```diff
@@ -461,4 +461,23 @@ variable "cloud_logs_data_storage" {
   }
   description = "A logs data bucket and a metrics bucket in IBM Cloud Object Storage to store your IBM Cloud Logs data for long term storage, search, analysis and alerting."
 }
+
+variable "enable_cloud_logs_as_target" {
+  description = "Whether to set IBM Cloud Logs be designated as a target for logs routing in a specific region."
+  type        = bool
+  default     = false
```

Comment:
same question as above, why not default to true?

Reply:
Not sure if there was any specific reason. Changed it to true.